src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Arm/Sve.PlatformNotSupported.cs

```diff
@@ -120,7 +120,65 @@ public new abstract class Arm64 : AdvSimd.Arm64
         /// </summary>
         public static unsafe Vector<ulong> CreateTrueMaskUInt64([ConstantExpected] SveMaskPattern pattern = SveMaskPattern.All) { throw new PlatformNotSupportedException(); }
 
+        ///  ConditionalSelect : Conditionally select elements
 
+        /// <summary>
```

Comment:
nit: It's preferred to order these "alphabetically" as well, since that's what tooling will do in various places.

This is done based on the type name, not the language keyword:
* `byte` (Byte), `double` (Double), `short` (Int16), `int` (Int32), `long` (Int64), `nint` (IntPtr), `sbyte` (SByte), `float` (Single), `ushort` (UInt16), `uint` (UInt32), `ulong` (UInt64), nuint (UIntPtr)

Reply:
@a74nh - do you mind fixing the tool to generate these alphabetically?